app/workers/periodic_task_reminder.rb

```diff
@@ -0,0 +1,11 @@
+require "sidekiq-scheduler"
+
+class PeriodicTaskReminder
+  include Sidekiq::Worker
+
+  def perform
+    Measure.tasks.with_pending_reminders.each do |measure|
+      # TODO: Send periodic reminder
```

Comment:
given the previous work we should probably use clockwork for this and not sidekiq (that is unless there are good reasons to do so), see https://github.com/dumparkltd/advocacy-tracker-server/issues/48#issuecomment-1253362406